src/main/java/wooteco/subway/path/domain/fare/distance/OverFiftyStrategy.java

```diff
@@ -0,0 +1,16 @@
+package wooteco.subway.path.domain.fare.distance;
+
+import static wooteco.subway.path.domain.fare.distance.TenToFiftyStrategy.MAX_DISTANCE;
+
+public class OverFiftyStrategy implements DistanceStrategy {
+    private static final int UNIT = 8;
+    private static final int UNIT_FARE = 100;
+
+    @Override
+    public int calculate(int distance) {
+        TenToFiftyStrategy strategy = new TenToFiftyStrategy();
+        int underFiftyFare = strategy.maxFare();
+        distance -= MAX_DISTANCE;
+        return underFiftyFare + (int) ((Math.ceil((distance - 1) / UNIT) + 1) * UNIT_FARE);
```

Comment:
```suggestion
        return underFiftyFare + (int) ((Math.ceil((distance - 1.) / UNIT) + 1) * UNIT_FARE);
```

정수끼리의 나누기 연산은 소수점이 사라질 수 있기 때문에 유의해주세요!

Reply:
오 이렇게 하면 되는군요!! 감사합니다 🤗